benchmarks/src/std/containers/sequences/vector.bool/move/test.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <benchmark/benchmark.h>
+//
+#include <algorithm>
+#include <functional>
+#include <random>
+#include <vector>
+
+using namespace std;
+
+inline static vector<bool> createRandomVector(const size_t size) {

```

Comment:
Does `inline` have an effect here? I guess we don't build with `/Ob1`, so it should not.

Reply:
When automatic inlining is enabled, the MSVC optimizer currently considers the presence of the `inline` keyword (explicitly or implicitly) to be a small hint, so it can have an effect.